lib/success.js

```diff
@@ -250,3 +291,24 @@ export default async function success(pluginConfig, context, { Octokit }) {
     throw new AggregateError(errors);
   }
 }
+
+export function loadAssociatedPRsQuery (shas) {
+  return `#graphql
+    query getAssociatedPRs($owner: String!, $repo: String!) {
+      repository(owner: $owner, name: $repo) {
+        ${shas.map((sha) => {
+          return `"${sha.slice(0, 6)}": object(oid: ${sha}) {
```

Comment:
node aliases have to begin with a letter, but most shas begin with a number

```suggestion
          return `"commit${sha.slice(0, 6)}": object(oid: ${sha}) {
```